config/express.server.config.ts

```diff
@@ -27,9 +30,9 @@ module.exports = function() {
     });
 
     // Configure HTTP routes
-    require('../routes/db.server.routes.js')(app);
-    require('../routes/forum.server.routes.js')(app);
-    require('../routes/user.server.routes.js')(app);
+    dbServerRoutes(app);
+    forumServerRoutes(app);
+    userServerRoutes(app);
```

Comment:
is `dbServerRoutes` etc usually put into a reexports file so that you can do the imports on a single line? eg: as a file `routes/routes.module.ts`